projects/1.18/assets/ydms-wizarding-world-of-harry-potter/harrypotter/lang/zh_cn.json

```
@@ -0,0 +1,117 @@
+{
+"effect.harrypotter.wingardium_leviosa_effect": "羽加迪姆·勒维奥萨",
+"screen.harrypotter.hotbar": "快捷栏",
+"screen.harrypotter.spellbook": "法术书",
+"screen.harrypotter.expleft": "剩余经验：",
+"screen.harrypotter.plus": "+",
+"screen.harrypotter.lvl": "等级",
+"screen.harrypotter.accio": "飞来咒",

```

```suggestion
"screen.harrypotter.accio": "飞来飞去",
```
`Accio`仅作`飞来`，前（后）接名词；但如果将`Accio`视作咒文整体的话，就应当译作`飞来飞去`
例：`Accio Broomstick` = `扫帚飞来`

---

但是实际效果来讲是打到什么就让什么`飞来`，所以`飞来飞去`应该不合适，不符合实际表现

---

实际上`飞来飞去`也是个传统错译了（因为没有`飞去`这部分）
我是坚持`飞来飞去`的，妥协一下的话`速速飞来`也可以